### PR TITLE
Add user facing documentation and mkdoc

### PR DESCRIPTION
**Pull request by Allda**
The mkdoc is used to generate a dynamic pages with a user documentation. The documentation is common for ISV and community operators. The docs will server as a official doc for operator users.

The mkdoc will run whenever a doc is updated and will be released to Github hosted environment.

JIRA: ISV-4507

### CODE REVIEW

**Comment by Allda** [comment]
The doc page is available at https://redhat-openshift-ecosystem.github.io/operator-pipelines/

File: docs/users/contributing-via-pr.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - Operator versioning strategy
+1. Verify tests and fix problems, if possible
+1. Ask for help in the PR in case of problems
+
+
+## Pull request
+
+When a pull request is created, a number of tests are executed via community hosted pipeline. One can see the results in the comment section of conversation tab.
+
+![PR](img/op_test_pr.png)
```
Comment (mantomas) [inline docs/users/contributing-via-pr.md:21]: The img directory is one level up, so this (and other examples within the .md files) needs to be referenced as `../img/..`

**Review by mantomas — APPROVED** [review]
Some nitpics, but overall LGTM

**Review by haripate — APPROVED** [review]
LGTM

**Pull request merged**